content/en/security/application_security/api-inventory/_index.md

```diff
@@ -0,0 +1,106 @@
+---
+
+## Overview
+
+API Security Inventory monitors your API traffic to provide visibility into the security posture of your APIs, including:
+
+- **Authentication**: type of authentication used (e.g. Basic Auth, API key, etc.)
+- **Public Exposure**: Whether the API is processing traffic from the internet
+- **Production status**: if the API is running in production environment
+- **Sensitive data flows**: Sensitive data handled by the API and flows between APIs.
+- **Attack Exposure**: if the endpoint is targeted by attacks (powered by [Application Threat Management][2])
+- **Vulnerabilities**:  if the endpoint contains vulnerability (powered by [Application Vulnerability Management][3])
+
+Using the API Security Inventory you can:
+
+- see at a glance your publicly exposed endpoints, which are not requiring authenticated requests
+- see at a glance your publicly exposed endpoints, which are handling sensitive data, such as payment card numbers
+- see which endpoints are at risk, and pivot directly into the [Threat Monitoring and Protection](/security/application_security/threats/) service for further investigation or response
+
+{{< img src="security/application_security/api/welcome.png" alt="API Security Inventory main page">}}
+
+## How to Enable
+
+If your library is configured to use [remote configuration][1], then the following tracer versions
+are compatible without any additional configuration.
+
+|Technology|Minimum version|
+|----------|----------|
+|Python    | v2.1.6   |
+
+## How Does it Work?
+
+API Inventory leverages the datadog tracing library with [ASM enabled](/security/application_security/enabling/) to gather security metadata about API traffic, including the API schema, types of sensitive data processed and the authentication scheme.
+
+
+- [Threat Monitoring and Protection](/security/application_security/threats/), and
+- [Vulnerability Management](/security/application_security/vulnerability_management/)
+
+API Inventory Security leverages [remote configuration][1] to manage and configure scanning rules detecting sensitive data and authentication.
+
+For each endpoint we calculate the risks outlined below.
+
+### Service in Production
+
+The `env` tag is checked for patterns that frequently represent non-production environments. For example, detecting `dev`, `alpha`, `beta`, `sandbox`, and similar values, will mark the environment as a non-production one.
+
+All other environments are marked as production.
+
+### Service Under Attack
+
+This risk is detected for API endpoints that have experienced [attacks](/security/application_security/threats/) in the last week.
+
+### Processing Sensitive Data
+
+[ASM](/security/application_security/threats/) matches known patterns for sensitive data in API requests. If anything matches, that endpoint is tagged with the type of sensitive data processed.
+
+<div class="alert alert-info">
+Sensitive data scanning in API responses is planned, but is not currently available.
```

Comment:
```suggestion
```
I think we should remove this as a callout and add it as a **Note** below.

Reply:
I removed this, and added it as a note to the "How to enable" table. Because different technologies will have different support levels.

Currently we have python listed, which supports both requests and responses. At some point we'll support ruby with only requests, etc.